offchain/state-server/Cargo.toml

```
@@ -9,8 +9,9 @@ name = "cartesi-rollups-state-server"
 path = "src/main.rs"
 
 [dependencies]
+log = { path = "../log" }
 types = { path = "../types" }
-
+clap = { workspace = true, features = ["derive", "env"] }
```

Keeping them in alphabetic order:
```suggestion
log = { path = "../log" }
```